statemachine/aws-stepfunctions-statemachine.json

```diff
@@ -205,7 +205,8 @@
     },
     "delete": {
       "permissions": [
-        "states:DeleteStateMachine"
+        "states:DeleteStateMachine",
+        "states:DescribeStateMachine"
```

Comment:
nit: let's also capture that we are documenting this and the motivation

Reply:
Updated the PR description to capture that we're looking into whether these permissions are referenced in any sort of public documentation.